docs/content/users/extend/custom-commands.md

```diff
@@ -186,6 +204,14 @@ The following fields can be used for a flag definition:
 * `NoOptDefVal`: default value, if the flag is on the command line without any options
 * `Annotations`: used by cobra.Command Bash autocomplete code (see <https://github.com/spf13/cobra/blob/main/site/content/completions/bash.md>)
 
+### "AutocompleteTerms" Annotation
```

Comment:
```suggestion
### `AutocompleteTerms` Annotation
```

With this you won't need it in spellchecklist any more

Reply:
That would be inconsistent with all of the other annotation headings... I'd lean towards having them all either all using backticks or all using double quotes, rather than mixing and matching.

That said, happy to make whatever change you think is appropriate - I'll leave as is for now, please confirm if you want just this one in backticks, all in backticks, or leave in double quotes.

Reply:
It's not that important, but `AutoCompleteTerms` is not a word, should not be spell-checked, and it's an "annotation". The others are probably wrong :) But perhaps they also happen to be words.

Reply:
I've swapped this one instance to backticks. I'll probably open a new issue in a few minutes to standardise it so they're all using backticks. Consistency will both make it easier for contributers in the future (since they'll be able to just copy the existing formatting without running into this issue) and remove any possible confusion from users (having one different from the rest might make it seem like there's something special or different about its usage or syntax or etc when there isn't)